metricbeat/docs/modules/sql.asciidoc

```diff
@@ -227,6 +227,51 @@ for each row. For example, this event is created for the first row:
 }
 ----
 
+*Oracle Database Connection Pre-requisites*
```

Comment:
Thanks for adding these notes.

They are added between a set of examples, could you please reorganize the docs so examples and these instructions are not mixed?